rs/ethereum/cketh/testnet/dfx.json

```diff
@@ -28,6 +28,15 @@
       "candid": "../../../rosetta-api/icrc1/index-ng/index-ng.did",
       "shrink": false
     },
+    "archive": {
+      "type": "custom",
+      "build": [
+        "bazel build //rs/rosetta-api/icrc1/archive:archive_canister_u256.wasm.gz"
+      ],
+      "wasm": "../../../../bazel-bin/rs/rosetta-api/icrc1/archive/archive_canister_u256.wasm.gz",
+      "candid": "../../../rosetta-api/icrc1/archive/archive.did",
+      "shrink": false
```

Comment:
Out of curiosity, why is `shrink` set to `false`?

Reply:
Haha, this I also initially asked myself and needed some time to find back the [original answer](https://gitlab.com/dfinity-lab/public/ic/-/merge_requests/16393#note_1671301766) from Roman:

> Because Bazel produces already shrunken canisters.

Plus, consistency, since all canisters declared in this file also use `"shrink": false`.